### DISCUSSION TITLE
Request support for the ISO 8601 standard 24:00:00 time format in .net

### DISCUSSION THREAD

**Discussion by choby** [dotnet/runtime#96537] [Ideas]
I am writing to request support for the ISO 8601 standard 24:00:00 time format in .net. Currently, this format is not supported, and I believe that adding support for it would be beneficial for developers who need to represent midnight in their applications.

I understand that the current standard for representing midnight is 00:00:00, but the ISO 8601 standard allows for 24:00:00 as an alternative representation. This format is used in some industries, such as aviation, and it would be helpful to have support for it in .net.

**Comment by gfoidl** [comment]
See also https://github.com/dotnet/runtime/issues/96536
